contracts/BSCValidatorSet.sol

```diff
@@ -837,14 +890,25 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
         validator.BBCFeeAddress = iter.next().toAddress();
       } else if (idx == 3) {
         validator.votingPower = uint64(iter.next().toUint());
-        success = true;
+        if (!finalityUpdated) {
```

Comment:
Please delete this, just mark `success = true` is enough.

Reply:
Fixed